### ISSUE TITLE
Update to actions that use recent Node.js

### ISSUE THREAD

**Issue by tgeoghegan** [divviup/janus#684]
On a [recent run of ci-build](https://github.com/divviup/janus/actions/runs/3331793334), GitHub Actions posted a warning:

>Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: actions-rs/toolchain, arduino/setup-protoc

https://github.com/actions-rs/toolchain/issues/219 tracks actions-rs/toolchain adopting a newer node.

https://github.com/arduino/setup-protoc/issues/45 tracks arduino/setup-protoc adopting a newer node.

I believe we're blocked here until the two issues above are resolved. This might be a problem since actions-rs/toolchain doesn't seem to be maintained.

**Comment by tgeoghegan** [comment]
https://github.com/divviup/janus/pull/685 moves us off actions-rs/toolchain altogether.

**Comment by RainbowMango** [comment]
+1
Just echo the warning message here:
```
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: 

arduino/setup-protoc
```

**Comment by RainbowMango** [comment]
cc the maintainer for help @divergentdave

**Comment by tgeoghegan** [comment]
We're aware of the problem. We're blocked on either https://github.com/arduino/setup-protoc/issues/45 being resolved or finding an alternate means of installing `protoc` in CI.

**Comment by divergentdave** [comment]
This was resolved in #1407, which removed our need for protoc. I spot checked each of our workflows, and we're not seeing these warnings anymore.
